lib/stat_tracker.rb

```diff
@@ -16,17 +16,4 @@ class StatTracker
   def self.from_csv(locations)
     stat_tracker = StatTracker.new(locations)
   end
-
-  #calls method in game stats, passes games location to be accessed
```

Comment:
I like moving these to another a different file in order to keep stat_tracker.rb short